images/runtime/.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
Maybe a sub directory of `contrib/packaging/docker` instead?

Reply:
IMO `images/xxx/` is a bit easier to type, but my main concern here would be to organise all of the dockerfiles consistently under some path rather than spread across different locations.

Reply:
Images are primary packaging format that is currently supported, it's mission-critical, so it deserves a top-level directory, which also make it easy to find.
A new contributor, or perhaps a user, who is wondering how are the Cilium images built, should be able to look in the repo, find `images/` directory and look no further.
Personally, when I see `contrib`, I don't look there as to me it stands for "collection of assorted bits that couldn't go anywhere else".

Reply:
> organise all of the dockerfiles consistently under some path rather than spread across different locations.

Definitely one of the main goals for me, unfortunately cannot touch the existing ones just yet, but hopefully can get rid of some of them in 1.9.